moto/cloudwatch/models.py

```diff
@@ -44,18 +60,127 @@ def __init__(self, namespace, name, value, dimensions):
                                      'value']) for dimension in dimensions]
 
 
+class MetricDatumT(BaseModel):
+
+    def __init__(self, namespace, data):
+        self.namespace = namespace
+        self.data = data
+
+
+class MetricDatumv2(BaseModel):
+
+    def __init__(self, namespace, metricdata):
+        self.Namespace = namespace
+    # parse the metricdata object
+
+        self.MetricData = [
```

Comment:
Same here about the uppercase. `Metric` is great as a class name but I'd recommend `self.metric_data = Metric(...`